containers/hub-xmlrpc-api-image/proxy-helm.changes.cbosdonnat.container-push

```diff
@@ -0,0 +1 @@
+- Simplify publishing to openbuildservice
```

Comment:
```suggestion
- Simplify publishing to Open Build Service
```

Reply:
oops... the file name needs adjustment too. Copying isn't a good idea here